plotter/heat.go

```diff
@@ -179,15 +179,15 @@ func (h *HeatMap) DataRange() (xmin, xmax, ymin, ymax float64) {
 		xmin = -0.5
 	default:
 		xmax = (3*h.GridXYZ.X(c-1) - h.GridXYZ.X(c-2)) / 2
```

Comment:
I wish the original author had left a sensible comment about where they got the 3 magic constant from. If I find out where he lives, so help me...